### PR TITLE
Manual updates for 3.0

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: demo/document.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-bundle.js"></script>
```
Comment (e111077) [inline demo/document.html]: Please change bundle to loader

File: demo/scrolling-region.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-bundle.js"></script>
```
Comment (e111077) [inline demo/scrolling-region.html]: please change bundle to loader

**Review by e111077 — CHANGES_REQUESTED** [review]

**Review by e111077 — CHANGES_REQUESTED** [review]
missing travis changes

**Review by e111077 — APPROVED** [review]

**Review by dfreedm — COMMENTED** [review]
Travis changes LGTM

**Pull request merged**